public/app/core/internationalization/index.tsx

```diff
@@ -56,7 +56,7 @@ export function changeLanguage(locale: string) {
 }
 
 export const Trans: typeof I18NextTrans = (props) => {
-  return <I18NextTrans {...props} />;
+  return <I18NextTrans shouldUnescape {...props} />;
```

Comment:
Can we add some unit tests to catch potential changes in i18next here? It should also assert your desired behaviour (that prometheus selectors render correctly)

It doesn't need to be super thorough, but just assert that things like HTML is still being escaped by either react or i18next :)

Reply:
Good idea! I added 2 test cases (rendering `<script></script` with and without `shouldUnescape={false}`). Correct Prometheus selectors rendering is asserted in `public/app/features/explore/Table/TableContainer.test.tsx`